misc/AL2023-Dockerfile

```diff
@@ -19,6 +19,12 @@ RUN dnf install -y \
     automake \
     autoconf
 
+# Some basic tools that are typical during builds
+RUN dnf install -y --allowerasing \
```

Comment:
What's the reason for `--allowerasing`? This looks suspicious. It either needs to be removed, or we need a comment why this is important.

Reply:
I will add a comment. gnupg2 conflicts with the minimal package that is already available, but does not ship a binary required to run rpmbuild -bp on e.g. the xz package.